src/deployer-watch/deployer-watch.js

```diff
@@ -0,0 +1,55 @@
+const { Finding, FindingSeverity, FindingType } = require('forta-agent');
+
+const addressList = require('./deployer-watch.json');
+const config = require('../../agent-config.json');
+
+const whitelist = {};
+(Object.keys(addressList)).forEach((a) => { whitelist[a.toLowerCase()] = true; });
+const deployerAddress = Object.keys(whitelist)[0];
```

Comment:
Probably a minor concern for these handlers, but consider placing this assignment inside the `.forEach()` in the previous statement.

On this line we are taking the first address (index 0) because we happen to know that the first entry in the file `deploy-watch.json` has the value `Deployer`.  However, if that file is ever modified and someone places an entry before the Deployer key/value pair, we are now erroneously grabbing whatever is in the first location.

Checking the value inside the `.forEach()` would remove the implicit knowledge present when we grab the entry at index 0.